src/app/service/tool-group/tool-group.service.ts

```diff
@@ -278,4 +279,38 @@ export class ToolGroupService extends AbstractService {
       })
       .catch(this.handleError);
   }
+
+  getToolGroupSuggestions(
+    countries: string[],
+    languages: string[],
+    confidence: string[],
+    openness: string[],
+  ): Promise<Resource[]> {
+    let filter = '';
+
+    const createFilters = (items: string[], filterString) => {
+      const filters = items.reduce(
+        (result: string, currentItem: string) =>
+          result + `${filterString}=${currentItem}&`,
+        '',
+      );
+      filter += filters;
+    };
+
+    createFilters(countries, 'filter[country]');
+    createFilters(languages, 'filter[language][]');
+    createFilters(openness, 'filter[openness]');
+    createFilters(confidence, 'filter[confidence]');
```

Comment:
language is the only filter that can send multiple values for the suggestions API, all the rest send 0 or 1 values

Reply:
on the mobile app we are going to be capturing a single country, openness & confidence values, then also capturing all languages they select

Reply:
Gotcha, I'll update the code